app/src/component/AppNavigator.js

```diff
@@ -174,6 +180,17 @@ class AppWithNavigationState extends React.Component {
     }
   }
   
+  _handleUrl = (evt) => {
```

Comment:
Does this work? I wouldn't think it has access to `this` since it's an arrow function

Reply:
Apparently, it does. The docs say it's experimental, however. It's called "public class fields syntax" on the page at https://reactjs.org/docs/handling-events.html.

I found out about it at https://medium.freecodecamp.org/react-binding-patterns-5-approaches-for-handling-this-92c651b5af56. It's the 5th item in the list.